### PR TITLE
Improve llvm syntax highlighting

### PR DESCRIPTION
**Pull request by Flakebi**
### Changes
- Numbers are allowed to have a leading + sign
- Hex floats can have a format specified after the 'x'
- Strings cannot escape quotes with backslashes, so a `\"` ends a string

### Checklist
- [x] Added markup tests, or they don't apply here because...
- [x] Updated the changelog at `CHANGES.md` (it’s a pretty small fix, not sure if this needs to be mentioned?)

### CODE REVIEW

File: src/languages/llvm.js [hunk 1]
```diff
@@ -109,12 +109,11 @@ export default function(hljs) {
       // another language than an actual comment
       hljs.COMMENT(/;\s*$/, null, { relevance: 0 }),
       hljs.COMMENT(/;/, /$/),
-      hljs.QUOTE_STRING_MODE,
       {
         className: 'string',
         variants: [
           // Double-quoted string
-          { begin: /"/, end: /[^\\]"/ },
+          { begin: /"/, end: /"/ },
```
Comment (joshgoebel) [inline src/languages/llvm.js:116]: If there is only one rules now then we no longer need the `variants` parent... and do you think while you're here perhaps we could add the actual escaping.  Is it not `\00` style, if I'm not mistaken?  IE, you'd just need a child inside `contains` that matched `\\\d\d` and scoped as `char.escape`.

Thoughts?

**Comment by joshgoebel** [comment]
> (it’s a pretty small fix, not sure if this needs to be mentioned?)

We mention almost all fixes. :)

**Comment by Flakebi** [comment]
Thanks! I fixed your comment and added a changelog entry. (diff of my changes: https://github.com/highlightjs/highlight.js/compare/be47761cb54d2bbeaf16006bb962df1b6f4eecd9..3ee1056de807099140def2e1e362cd6e2ae4853d)

**Review by joshgoebel — APPROVED** [review]

**Comment by joshgoebel** [comment]
| Filename | Size | Change |
| :--- | :---: | :---: |
| `./build/es/core.min.js` | 8 kB | +1 B (0%) |
| `./build/es/highlight.min.js` | 8 kB | +1 B (0%) |
| `./build/es/languages/llvm.min.js` | 1.44 kB | +7 B (0%) |
| `./build/highlight.min.js` | 8.04 kB | -1 B (0%) |
| `./build/languages/llvm.min.js` | 1.45 kB | +8 B (+1%) |

**Comment by joshgoebel** [comment]
@Flakebi Thanks so much!

**Pull request merged**